packages/gatsby-theme-vtex/src/sdk/product/useSummaryImage.ts

```diff
@@ -0,0 +1,24 @@
+import { useScaledImage } from '../img/useScaledImage'
+import {
+  SUMMARY_IMAGE_HEIGHT,
+  SUMMARY_IMAGE_HEIGHT_STR,
+  SUMMARY_IMAGE_WIDTH,
+  SUMMARY_IMAGE_WIDTH_STR,
+} from './constants'
+
+export const IMAGE_DEFAULT =
+  'https://tudoparasuaempresa.com.br/assets/img/!product-image.jpg'
```

Comment:
I think this would be the best opportunity to put this image in our file manager, what do you think?